.github/workflows/fun_tests.yml

```diff
@@ -0,0 +1,118 @@
+name: "Run Cypress"
+
+on:
+  workflow_call:
+    inputs:
+      sha:
+        required: true
+        type: string
+
+env:
+  # increment it in case if you need to reset cache
+  CACHE_NAME_PREFIX: v3
+  NODE: '14'
```

Comment:
node 16?

Reply:
that's a separate task, we have to update to node LTS version everywhere, but currently nothing works on 16+